components/infobox/wikis/simracing/infobox_person_player_custom.lua

```diff
@@ -6,10 +6,9 @@
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
 
-local Array = require('Module:Array')
+local GameAppearances = require('Module:GetGameAppearances')
```

Comment:
Can we make this info an Infobox Extension instead?

Reply:
How should I do it to achieved that? The appearance module [looked like this](https://liquipedia.net/simracing/Module:GetGameAppearances)

Reply:
https://github.com/Liquipedia/Lua-Modules/tree/main/components/infobox/extensions/commons

Reply:
ok, I'll make a separate PR for adding the module as commons, will set this simracing PR as draft for now